src/services/__tests__/fhrsApi.test.ts

```diff
@@ -1,161 +1,54 @@
-      '836583',
-      '997140',
-      '743833',
-      '988109',
-      '268454',
-      '964599',
-      '1011214',
-      '269163',
-      '841948',
-      '889450',
-      '268806',
-      '1015461',
-      '268453',
-      '929914',
-      '269262',
-      '268351',
-      '269202',
-      '268849',
-      '777807',
-      '268441',
-      '997130',
-      '1031734',
-      '661417',
-      '269447',
-      '269434',
-      '269449',
-      '268866',
-      '269448',
-      '269442',
-      '269441',
-      '269436',
-      '269437',
-      '269446',
-      '269438',
-      '269452',
-      '269433',
-      '955351',
-      '269493',
-      '268355',
-      '269260',
-      '269361',
-      '268548',
-      '268352',
-      '269402',
-      '681923',
-      '269142',
-      '268353',
-      '269391',
-      '661418',
-      '904891',
-      '268433',
-      '268667',
-      '1401890',
-      '268356',
-      '268993',
-      '268935',
-      '268934',
-      '268933',
-      '268435',
-      '269201',
-      '848730',
-      '268461',
-      '268410',
-      '668853',
-      '268447',
-      '269155',
-      '268450',
-      '1292333',
-    ];
+  it('should return a rating between 0 and 5', async () => {
+    const fetchJsonMock = jest
+      .spyOn(fetchModule, 'fetchJson')
+      .mockResolvedValue({
+        FHRSID: 423824,
+        ChangesByServerID: 0,
+        LocalAuthorityBusinessID: '13359',
+        BusinessName: 'Caffe Nero',
+        BusinessType: 'Restaurant/Cafe/Canteen',
+        BusinessTypeID: 1,
+        AddressLine1: '',
+        AddressLine2: '30 Monmouth Street',
+        AddressLine3: '',
+        AddressLine4: 'London',
+        PostCode: 'WC2H 9HA',
+        Phone: '',
+        RatingValue: '5',
+        RatingKey: 'fhrs_5_en-gb',
+        RatingDate: '2019-09-11T00:00:00',
+        LocalAuthorityCode: '506',
+        LocalAuthorityName: 'Camden',
+        LocalAuthorityWebSite: 'http://www.camden.gov.uk',
+        LocalAuthorityEmailAddress: 'foodsafety@camden.gov.uk',
+        scores: {
+          Hygiene: 0,
+          Structural: 0,
+          ConfidenceInManagement: 0,
+        },
+        SchemeType: 'FHRS',
+        geocode: {
+          longitude: '-0.1272279',
+          latitude: '51.5136148',
+        },
+        RightToReply: '',
+        Distance: null,
+        NewRatingPending: false,
+      });
 
-    fhrsIds.forEach(async (id) => {
-      const rating = await getEstablishmentRatingValue(parseInt(id, 10));
-      if (Number.isNaN(rating)) {
-        expect(rating).toBeNaN();
-        return;
-      }
-      expect(rating).toBeGreaterThanOrEqual(0);
-      expect(rating).toBeLessThanOrEqual(5);
-    });
+    const fhrsId = '269382';
+    const rating = await getEstablishmentRatingValue(parseInt(fhrsId, 10));
+
+    if (Number.isNaN(rating)) {
+      expect(rating).toBeNaN();
+      return;
+    }
+    expect(rating).toBeGreaterThanOrEqual(0);
+    expect(rating).toBeLessThanOrEqual(5);
```

Comment:
This should be enough:

```suggestion
    const rating = await getEstablishmentRatingValue(423824);
    expect(rating).toBe(5);
```